src/sanity.ts

```diff
@@ -0,0 +1,530 @@
+        },
+        {
+          name: "list",
+          description:
+            "Lists all the GraphQL endpoints deployed for this project",
+        },
+        {
+          name: "undeploy",
+          description: "Remove a deployed GraphQL API",
+        },
+      ],
+    },
+    {
+      name: "hook",
+      description: "Interact with hooks in your project",
+      subcommands: [
+        {
+          name: "attempt",
+          description: "Print details of a given webhook delivery attempt",
+        },
+        {
+          name: "create",
+          description: "Create a new hook for the given dataset",
+        },
+        {
+          name: "delete",
+          description: "Delete a hook within your project",
+        },
+        {
+          name: "list",
+          description: "List hooks for a given project",
+        },
+        {
+          name: "logs",
+          description: "List latest log entries for a given hook",
+        },
+      ],
+    },
+    {
+      name: "init",
+      description: "Initialize a new Sanity project or plugin",
+      options: [
+        {
+          name: ["-y", "--yes"],
+          description:
+            "Use unattended mode, accepting defaults and using only flags for choices",
+        },
+        {
+          name: "--project",
+          description: "Project ID to use for the studio",
+          args: {
+            name: "projectId",
+          },
+        },
+        {
+          name: "--organization",
+          description: "Organization ID to use for the project",
+          args: {
+            name: "organizationId",
+          },
+        },
+        {
+          name: "--dataset",
+          description: "Dataset name for the studio",
+          args: {
+            name: "dataset",
+          },
+        },
+        {
+          name: "--dataset-default",
+          description:
+            'Set up a project with a public dataset named "production"',
+        },
+        {
+          name: "--output-path",
+          description: "Path to write studio project to",
+          args: {
+            name: "path",
+          },
+        },
+        {
+          name: "--template",
+          description: 'Project template to use [default: "clean"]',
+          args: {
+            name: "template",
+          },
+        },
+        {
+          name: "--provider",
+          description: "Login provider to use",
+          args: {
+            name: "provider",
+          },
+        },
+        {
+          name: "--visibility",
+          description: "Visibility mode for dataset (public/private)",
+          args: {
+            name: "mode",
+          },
+        },
+        {
+          name: "--create-project",
+          description: "Create a new project with the given name",
+          args: {
+            name: "name",
+          },
+        },
+        {
+          name: "project-plan",
```

Comment:
Should have `--` in front?
```suggestion
          name: "--project-plan",
```